### PR TITLE
[type:feature] upgrade checkstyle plugin to 3.4.0

### PR DESCRIPTION
**Pull request by Aias00**
upgrade checkstyle plugin to 3.4.0

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-web/src/main/java/org/apache/shenyu/web/fallback/DefaultFallbackController.java [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
-    }
-    
-    /**
-     * Sentinel fall back object.
-     *
-     * @return the object
-     */
-    @GetMapping("/sentinel")
-    public Object sentinelFallBack() {
-        return ShenyuResultWrap.error(ShenyuResultEnum.SENTINEL_PLUGIN_FALLBACK, null);
```
Comment (moremind) [inline shenyu-web/src/main/java/org/apache/shenyu/web/fallback/DefaultFallbackController.java:60]: why rewrite this class?

File: shenyu-plugin/shenyu-plugin-security/shenyu-plugin-basic-auth/src/main/java/org/apache/shenyu/plugin/basic/auth/handle/BasicAuthPluginDataHandler.java [hunk 2]
```diff
@@ -1,72 +1,72 @@
...
-import org.apache.shenyu.common.enums.PluginEnum;
-import org.apache.shenyu.common.utils.GsonUtils;
-import org.apache.shenyu.common.utils.Singleton;
-import org.apache.shenyu.plugin.base.cache.CommonHandleCache;
-import org.apache.shenyu.plugin.base.handler.PluginDataHandler;
-import org.apache.shenyu.plugin.base.utils.BeanHolder;
-import org.apache.shenyu.plugin.base.utils.CacheKeyUtils;
-import org.apache.shenyu.plugin.basic.auth.config.BasicAuthConfig;
-import org.apache.shenyu.plugin.basic.auth.rule.BasicAuthRuleHandle;
-
```
Comment (moremind) [inline shenyu-plugin/shenyu-plugin-security/shenyu-plugin-basic-auth/src/main/java/org/apache/shenyu/plugin/basic/auth/handle/BasicAuthPluginDataHandler.java:37]: why rewrite this class?

**Review by moremind — CHANGES_REQUESTED** [review]

File: shenyu-plugin/shenyu-plugin-security/shenyu-plugin-basic-auth/src/main/java/org/apache/shenyu/plugin/basic/auth/handle/BasicAuthPluginDataHandler.java (hunk 2 above)
Comment (Aias00) [reply shenyu-plugin/shenyu-plugin-security/shenyu-plugin-basic-auth/src/main/java/org/apache/shenyu/plugin/basic/auth/handle/BasicAuthPluginDataHandler.java:37]: changed line separators from CRLF to LF, which is required by the checkStyle 
image

File: shenyu-web/src/main/java/org/apache/shenyu/web/fallback/DefaultFallbackController.java (hunk 1 above)
Comment (Aias00) [reply shenyu-web/src/main/java/org/apache/shenyu/web/fallback/DefaultFallbackController.java:60]: changed line separators from CRLF to LF, which is required by the checkStyle

image

**Pull request merged**